src/main/resources/config/application-kubernetes.yml

```diff
@@ -13,4 +13,4 @@
 spring:
   kafka:
     bootstrap-servers:
-      - foundation-kafka:9092
+      - alm-kafka:9092
```

Comment:
could you also update docs/ConfiguringKafka.md to reflect this change please

Reply:
Done